src/dynamics.coffee

```diff
@@ -173,7 +173,7 @@ getCurrentProperties = (el, keys) ->
           properties['transform'] = matrix.decompose()
       else
         v = style[key]
-        if !v? && svgProperties.contains(key)
+        if !v? && key != 'd' && svgProperties.contains(key)
```

Comment:
Are you sure that's what you want?
It seems like you want something like this instead:
```
if (!v? && svgProperties.contains(key)) || key == 'd'
```